src/useOneTrust.ts.jest

```diff
@@ -0,0 +1,100 @@
+import { act, renderHook } from '@testing-library/react';
```

Comment:
When I run test locally `npm test` I see as below. I am not sure if I am doing correctly 

```
> @tl-its-umich-edu/react-ga-onetrust-consent@1.0.0 test
> jest

No tests found, exiting with code 1
Run with `--passWithNoTests` to exit with code 0
In /Users/pushyami/git_tl_projects/react-ga-onetrust-consent
  12 files checked.
  testMatch: **/__tests__/**/*.[jt]s?(x), **/?(*.)+(spec|test).[tj]s?(x) - 0 matches
  testPathIgnorePatterns: /node_modules/ - 12 matches
  testRegex:  - 0 matches
Pattern:  - 0 matches
```

Reply:
oh I didn't realize the files in the `dist` directory need to be generated by typescript first before testing, I'll add that to the test command